packages/zoe/src/types-ambient.js

```diff
@@ -38,6 +36,8 @@
  * @property {Issuer<K, M>} issuer
  * @property {K} assetKind
  * @property {DisplayInfo<K>} [displayInfo]
- *
- * @typedef {AmountKeywordRecord} Allocation
+ */
+
+/**
+ * @typedef {Record<Keyword, import('@agoric/ertp/exported.js').AnyAmount>} Allocation
```

Comment:
Likewise, I'd like to understand the motivation for this change better.

Reply:
ditto